src/main/java/org/matsim/analysis/emissions/RunOfflineAirPollutionAnalysisByVehicleCategory.java

```diff
@@ -69,18 +73,23 @@ public static void main(String[] args) {
 			String rootDirectory = args[0];
 			if (!rootDirectory.endsWith("/")) rootDirectory = rootDirectory + "/";
 			
-			final String runDirectory = "public-svn/matsim/scenarios/countries/de/berlin/berlin-v5.4-10pct/output-berlin-v5.4-10pct/";	
-			final String runId = "berlin-v5.4-10pct";
+			final String runDirectory = "public-svn/matsim/scenarios/countries/de/berlin/berlin-v5.5-1pct/output-berlin-v5.5-1pct/";
+			final String runId = "berlin-v5.5.3-1pct";
 
-			final String hbefaFileCold = "shared-svn/projects/matsim-germany/hbefa/hbefa-files/v3.2/EFA_ColdStart_vehcat_2005average.txt";
-			final String hbefaFileWarm = "shared-svn/projects/matsim-germany/hbefa/hbefa-files/v3.2/EFA_HOT_vehcat_2005average.txt";
+			final String hbefaFileCold = "shared-svn/projects/matsim-germany/hbefa/hbefa-files/v4.1/EFA_ColdStart_Vehcat_2020_Average_withHGVetc.csv";
+			final String hbefaFileWarm = "shared-svn/projects/matsim-germany/hbefa/hbefa-files/v4.1/EFA_HOT_Vehcat_2020_Average.csv";
+
+			final String detailedFileCold = "shared-svn/projects/matsim-germany/hbefa/hbefa-files/v4.1/EFA_ColdStart_Concept_2020_detailed_perTechAverage_withHGVetc.csv";
+			final String detailedFileWarm = "shared-svn/projects/matsim-germany/hbefa/hbefa-files/v4.1/EFA_HOT_Concept_2020_detailed_perTechAverage.csv";
```

Comment:
If possible, please use the encrypted files, see e.g. https://github.com/matsim-scenarios/matsim-berlin/blob/5.5.x/src/test/java/org/matsim/analysis/RunOfflineAirPollutionAnalysisTest.java for an example 
and https://github.com/matsim-scenarios/matsim-berlin/pull/76/ for how to put in the decryption.

In the shared-svn there is a document with the information, which encrypted table correspond to which "normal" table.

Reply:
oh, now I see, that you have it already in the other file, you changed in this PR :) -> Please do it here analogously.